src/trivyHelper.ts

```diff
@@ -83,28 +96,55 @@ export function getSummary(trivyStatus: number): string {
     return `- ${summary}`;
 }
 
-function getVulnerabilityIds(trivyStatus: number): string[] {
-    let vulnerabilityIds: string[] = [];
+export function printFormattedOutput() {
+    let rows = [];
+    let titles = [TITLE_VULNERABILITY_ID, TITLE_PACKAGE_NAME, TITLE_SEVERITY, TITLE_DESCRIPTION];
+    rows.push(titles);
+
+    const vulnerabilities = getVulnerabilities();
```

Comment:
You missed sending `true` param

Reply:
That is intentional. In logs we want the duplicate `CVEs`, as we are showing the `package name` as well. So the same `CVEs` might be present for different packages.